crates/polars-core/src/utils/mod.rs

```diff
@@ -133,7 +133,11 @@ pub fn split_series(s: &Series, n: usize) -> PolarsResult<Vec<Series>> {
     split_array!(s, n, i64)
 }
 
-pub fn split_df_as_ref(df: &DataFrame, n: usize) -> PolarsResult<Vec<DataFrame>> {
+pub fn split_df_as_ref(
```

Comment:
This is technically a public API, though nothing uses it at least in Polars, just `split_df()`, so if backwards compat is an issue then this could be a new function and `split_df_as_ref` could be a wrapper. (Shame Rust has no default keyword args, makes this sort of change easier to be backwards compat.)